tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -81,12 +81,12 @@ def test_memory_exhaustion(self, duthosts, enum_rand_one_per_hwsku_hostname, loc
         # Waiting for SSH connection startup
         pytest_assert(self.check_ssh_state(localhost, dut_ip, SSH_STATE_STARTED, SSH_STARTUP_TIMEOUT),
                       'DUT {} did not startup'.format(hostname))
+        # Verify DUT is really rebooted: uptime should be later than the time when the test case started.
+        dut_uptime = duthost.get_up_time()
+        pytest_assert(dut_uptime > dut_datetime, "Device {} did not reboot".format(hostname))
```

Comment:
wait until this condition?

Reply:
Line 79 wait 480s for SSH disconnect.
Line 82 wait 600s for SSH reconnect.
I think that's long enough for DUT reboot. Is wait util still required for uptime checking?

Reply:
Per discussion, removed SSH dis/re-connect check. Added wait-until for uptime checking.